src/main/java/com/spikes2212/command/genericsubsystem/commands/MoveGenericSubsystemWithPIDAndFeedForward.java

```diff
@@ -17,7 +15,7 @@
  * @author Yuval Levy
  * @see GenericSubsystem
  */
-public class MoveGenericSubsystemWithPID extends CommandBase {
+public class MoveGenericSubsystemWithPIDAndFeedForward extends CommandBase {
```

Comment:
Unify this command with `MoveGenericSubsystemWithPID` in the way described above.